### ISSUE TITLE
我停止了思考

### ISSUE THREAD

**Issue by njsgdd10086** [thu-coai/EVA#78]
<img width="1644" alt="image" src="https://user-images.githubusercontent.com/103729896/216827337-1bf8a792-5f86-46b6-987b-4ef20e8a32ca.png">

**Comment by t1101675** [comment]
如果有问题请描述运行的环境和配置
如果没有问题了可以关闭 issue

**Comment by njsgdd10086** [comment]
Package                       Version
----------------------------- -------------
absl-py                       1.3.0
aiohttp                       3.8.3
aiosignal                     1.3.1
altair                        4.2.0
ansicon                       1.89.0
anyio                         3.6.2
apex                          0.1
appdirs                       1.4.4
APScheduler                   3.9.1.post1
argon2-cffi                   21.3.0
argon2-cffi-bindings          21.2.0
arrow                         1.2.3
astor                         0.8.1
asttokens                     2.1.0
astunparse                    1.6.3
async-timeout                 4.0.2
attrdict                      2.0.1
attrs                         22.1.0
audioread                     3.0.0
Babel                         2.11.0
backcall                      0.2.0
backports.functools-lru-cache 1.6.4
backports.zoneinfo            0.2.1
bce-python-sdk                0.8.74
beautifulsoup4                4.11.1
bleach                        5.0.1
blessed                       1.19.1
boto3                         1.26.60
botocore                      1.29.60
Brotli                        1.0.9
cachetools                    5.2.0
certifi                       2022.9.24
cffi                          1.15.1
chardet                       5.1.0
charset-normalizer            2.1.1
click                         8.1.3
cn2an                         0.5.19
colorama                      0.4.6
coloredlogs                   15.0.1
commonmark                    0.9.1
contourpy                     1.0.6
croniter                      1.3.8
cryptography                  39.0.0
cssselect                     1.2.0
cssutils                      2.6.0
cycler                        0.11.0
Cython                        0.29.23
datasets                      2.7.1
debugpy                       1.6.3
decorator                     5.1.1
deepdiff                      6.2.2
deepspeed                     0.3.9
defusedxml                    0.7.1
Deprecated                    1.2.13
dill                          0.3.6
dnspython                     2.2.1
einops                        0.6.0
email-validator               1.3.0
eng-to-ipa                    0.0.2
entrypoints                   0.4
et-xmlfile                    1.1.0
executing                     1.2.0
fastapi                       0.88.0
fastjsonschema                2.16.2
ffmpeg-python                 0.2.0
ffmpy                         0.3.0
ffmpy3                        0.2.4
filelock                      3.8.0
fire                          0.5.0
Flask                         2.0.2
flask-babel                   3.0.1
Flask-SQLAlchemy              2.5.1
flatbuffers                   22.11.23
fonttools                     4.38.0
frozenlist                    1.3.3
fsspec                        2022.11.0
future                        0.18.2
gevent                        22.10.2
geventhttpclient              2.0.8
google-auth                   2.14.1
google-auth-oauthlib          0.4.6
gradio                        3.16.1
greenlet                      2.0.1
grpcio                        1.41.0
h11                           0.14.0
h5py                          3.7.0
httpcore                      0.16.3
httptools                     0.5.0
httpx                         0.23.1
huggingface-hub               0.10.1
humanfriendly                 10.0
idna                          3.4
imageio                       2.22.4
imgaug                        0.4.0
importlib-metadata            5.0.0
importlib-resources           5.10.0
indic-transliteration         2.3.40
inflect                       6.0.2
inquirer                      3.1.1
install                       1.3.5
ipykernel                     6.17.1
ipython                       8.6.0
ipython-genutils              0.2.0
ipywidgets                    8.0.2
itchat                        1.3.10
itchat-uos                    1.4.1
itsdangerous                  2.1.2
jaconv                        0.3
jamo                          0.4.1
jedi                          0.18.1
jieba                         0.42.1
Jinja2                        3.1.2
jinxed                        1.2.0
jmespath                      1.0.1
joblib                        1.2.0
jsonschema                    4.17.0
jupyter                       1.0.0
jupyter_client                7.4.5
jupyter-console               6.4.4
jupyter_core                  5.0.0
jupyter-server                1.23.2
jupyterlab-pygments           0.2.2
jupyterlab-widgets            3.0.3
kiwisolver                    1.4.4
ko-pron                       1.3
labelImg                      1.8.6
librosa                       0.9.1
lightning                     1.8.6
lightning-cloud               0.5.15
lightning-utilities           0.5.0
linkify-it-py                 1.0.3
llvmlite                      0.31.0
lmdb                          1.4.0
lxml                          4.9.1
Markdown                      3.4.1
markdown-it-py                2.1.0
MarkupSafe                    2.1.1
matplotlib                    3.6.2
matplotlib-inline             0.1.6
mdit-py-plugins               0.3.3
mdurl                         0.1.2
mistune                       2.0.4
more-itertools                9.0.0
MouseInfo                     0.1.3
mpmath                        1.2.1
multidict                     6.0.2
multiprocess                  0.70.14
mypy-extensions               0.4.3
nbclassic                     0.4.8
nbclient                      0.7.0
nbconvert                     7.2.5
nbformat                      5.7.0
nest-asyncio                  1.5.6
networkx                      2.8.8
ninja                         1.11.1
nltk                          3.8.1
notebook                      6.5.2
notebook_shim                 0.2.2
ntchat                        0.1.20
num-thai                      0.0.5
numba                         0.48.0
numpy                         1.21.6
nvidia-dllogger               0.1.0
nvidia-pyindex                1.0.9
oauthlib                      3.2.2
onnx                          1.13.0
onnxruntime-gpu               1.13.1
OpenCC                        1.1.1
opencv-contrib-python         4.7.0.68
opencv-python                 4.6.0.66
openpyxl                      3.0.10
opt-einsum                    3.3.0
ordered-set                   4.1.0
orjson                        3.8.3
packaging                     21.3
paddle-bfloat                 0.1.7
paddleocr                     2.6.1.2
paddlepaddle-gpu              2.4.1.post117
pandas                        1.5.2
pandocfilters                 1.5.0
parso                         0.8.3
pdf2docx                      0.5.6
pickleshare                   0.7.5
Pillow                        9.3.0
pip                           23.0
pkgutil_resolve_name          1.3.10
platformdirs                  2.5.4
pooch                         1.6.0
praat-parselmouth             0.4.1
premailer                     3.10.0
proces                        0.1.3
prometheus-client             0.15.0
prompt-toolkit                3.0.32
protobuf                      3.20.3
psutil                        5.9.4
pure-eval                     0.2.2
pyarrow                       10.0.1
pyasn1                        0.4.8
pyasn1-modules                0.2.8
PyAutoGUI                     0.9.53
pybind11                      2.10.3
pyclipper                     1.3.0.post4
pycparser                     2.21
pycryptodome                  3.16.0
pycwt                         0.3.0a22
pydantic                      1.10.2
pydub                         0.25.1
pyee                          9.0.4
PyExecJS                      1.5.1
pyflowchart                   0.2.3
PyGetWindow                   0.0.9
Pygments                      2.13.0
PyJWT                         2.6.0
pykakasi                      2.2.1
pyloudnorm                    0.1.0
PyMsgBox                      1.0.9
PyMuPDF                       1.20.2
PyMySQL                       1.0.2
pynvml                        11.4.1
pyparsing                     3.0.9
pyperclip                     1.8.2
pypinyin                      0.47.1
pypng                         0.20220715.0
PyQRCode                      1.2.1
PyQt5                         5.15.7
PyQt5-Qt5                     5.15.2
PyQt5-sip                     12.11.0
pyre-extensions               0.0.23
pyreadline3                   3.4.1
PyRect                        0.2.0
pyrsistent                    0.19.2
PyScreeze                     0.1.28
python-dateutil               2.8.2
python-docx                   0.8.11
python-dotenv                 0.21.0
python-editor                 1.0.4
python-multipart              0.0.5
python-rapidjson              1.9
pytorch-lightning             1.8.6
pytweening                    1.0.4
pytz                          2022.7.1
pytz-deprecation-shim         0.1.0.post0
PyWavelets                    1.4.1
pywin32                       305
pywinpty                      2.0.9
PyYAML                        6.0
pyzmq                         24.0.1
qq-bot                        0.8.2
qq-botpy                      1.1.1
qrcode                        7.3.1
qtconsole                     5.4.0
QtPy                          2.3.0
rapidfuzz                     2.13.7
rarfile                       4.0
readchar                      4.0.3
regex                         2022.10.31
requests                      2.28.1
requests-oauthlib             1.3.1
resampy                       0.3.1
responses                     0.18.0
rfc3986                       1.5.0
rich                          12.6.0
roman                         3.3
rsa                           4.9
s3transfer                    0.6.0
scikit-image                  0.19.3
scikit-learn                  1.1.3
scipy                         1.9.3
seaborn                       0.12.1
Send2Trash                    1.8.0
sentencepiece                 0.1.97
setuptools                    65.5.1
shapely                       2.0.0
six                           1.16.0
sklearn                       0.0.post1
sniffio                       1.3.0
soundfile                     0.11.0
soupsieve                     2.3.2.post1
SQLAlchemy                    1.4.45
stack-data                    0.6.1
starlette                     0.22.0
starsessions                  1.3.0
sympy                         1.11.1
tensorboard                   2.11.0
tensorboard-data-server       0.6.1
tensorboard-plugin-wit        1.8.1
tensorboardX                  1.8
termcolor                     2.2.0
terminado                     0.17.0
threadpoolctl                 3.1.0
tifffile                      2022.10.10
tinycss2                      1.2.1
tokenizers                    0.13.2
toml                          0.10.2
toolz                         0.12.0
torch                         1.13.1+cu117
torchaudio                    0.13.1+cu117
torchcrepe                    0.0.17
torchvision                   0.14.1+cu117
tornado                       6.2
tqdm                          4.64.1
traitlets                     5.5.0
transformers                  4.24.0
tritonclient                  2.22.0
typer                         0.7.0
typing_extensions             4.4.0
typing-inspect                0.8.0
tzdata                        2022.7
tzlocal                       4.2
uc-micro-py                   1.0.1
ujson                         5.6.0
Unidecode                     1.3.6
urllib3                       1.26.12
uvicorn                       0.20.0
visualdl                      2.5.0
watchfiles                    0.18.1
wcwidth                       0.2.5
webencodings                  0.5.1
webrtcvad-wheels              2.0.11.post1
websocket-client              1.4.2
websockets                    10.4
Werkzeug                      2.2.2
wheel                         0.38.4
whisper                       1.0
widgetsnbextension            4.0.3
wrapt                         1.14.1
x2paddle                      1.4.0
xformers                      0.0.16rc424
xxhash                        3.1.0
yarl                          1.8.1
zipp                          3.10.0
zope.event                    4.6
zope.interface                5.5.2

python版本是3.8
cuda版本11.7
系统是windous11

**Comment by t1101675** [comment]
我们没有在 windows 系统上做过测试，可以试试按照 readme 中的步骤使用 docker

**Comment by njsgdd10086** [comment]
有计划做个docker使用的教程吗？docker用的不多，push完run了以后就不知道怎么操作了

**Comment by njsgdd10086** [comment]
![( MK(AMSQ8$_~$HK52NJWLW](https://user-images.githubusercontent.com/103729896/217002860-42d7fd87-9871-48fb-a731-90ff00a2a85e.png)
之前忘记说了，src/tokenization_eva.py的第35行没有指定编码，跑的时候会报错

**Comment by t1101675** [comment]
> 有计划做个docker使用的教程吗？docker用的不多，push完run了以后就不知道怎么操作了

抱歉这不在本 repo 的范围之内了。网上有很多 docker 相关的[教程](https://docker-curriculum.com/)，可以参考 。

**Comment by njsgdd10086** [comment]
啊，好的感谢了
